dev/io.openliberty.grpc.client.1.0.jakarta.thirdparty/bnd.bnd

```diff
@@ -0,0 +1,75 @@
+#*******************************************************************************
+# Copyright (c) 2020, 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+grpcVersion=1.38.1
+nettyVersion=4.1.68.Final
+
+Bundle-Name: Open Liberty gRPC Client Third Party API
+Bundle-SymbolicName: io.openliberty.grpc.client.1.0.javaee.thirdparty
+Bundle-Description: Open Liberty gRPC Client Third Party API; Jakarta enabled
+
+-pom: artifactid=io.openliberty.grpc.client.1.0.jakarta.thirdparty
+
+jakartaeeMe: true
+jakartaFinalJarName: io.openliberty.grpc.client.1.0.jakarta.thirdparty.jar
+jakartaFinalBundleId: io.openliberty.grpc.client.1.0.jakarta.thirdparty
+
+# Only publish the Jakarta one
+publish.wlp.jar.include: io.openliberty.grpc.client.1.0.jakarta.thirdparty.jar
+
+# Using version=! in order to not have a version attached to the import for packages that were removed
+# from Java after Java 8.  Doing this keeps the import like before Java 11 support. It will get the 
+# packages from Java when using Java 8 or earlier and from the new shipped bundles for Java 9 and later.
+Import-Package: \
+  !com.aayushatharva.brotli4j*,\
+  !com.github.luben.zstd,\
+  !com.google.code.gson,\
+  !org.checkerframework,\
+  !com.google.protobuf.nano,\
+  !com.jcraft.jzlib,\
+  !com.ning.*,\
+  !com.oracle.svm.core.annotate,\
+  !io.netty.internal.tcnative,\
+  !lzma.*,\
+  !net.jpountz.*,\
+  !org.apache.log4j.*,\
+  !org.apache.logging.*,\
+  !org.bouncycastle.*,\
+  !org.conscrypt,\
+  !org.eclipse.jetty.*,\
+  !org.jboss.*,\
+  !reactor.blockhound.*,\
+  !sun.*,\
+  javax.annotation;version=!,\
+  *
+
+Export-Package:\
+  io.grpc.netty;version=${grpcVersion}, \
+  io.netty.handler.ssl;version=${nettyVersion}
```

Comment:
package is already exported by `io.openliberty.io.netty.ssl` so why not use that?

Reply:
Spoke to Bill Lucy, these bundles are third-party and `io.openliberty.io.netty.ssl` is an internal bundle. Customer would not be able to use this package otherwise.